tests/integration/dhcp_test.py

```diff
@@ -192,6 +193,126 @@ def test_dhcp_for_bond_with_ip_address_and_slave(dhcp_env,
     assertlib.assert_state(desired_state)
 
 
+def test_ipv4_dhcp_ignore_gateway(dhcp_env):
+    desired_state = statelib.show_only((DHCP_CLI_NIC,))
+    dhcp_cli_desired_state = desired_state[INTERFACES][0]
+    dhcp_cli_desired_state['state'] = 'up'
+    dhcp_cli_desired_state['ipv4']['enabled'] = True
+    dhcp_cli_desired_state['ipv4']['dhcp'] = True
+    dhcp_cli_desired_state['ipv4']['ignore-auto-gateway'] = True
+    dhcp_cli_desired_state['ipv6']['enabled'] = True
```

Comment:
Why is this enabled if the test is about ipv4?

Reply:
Once interface up, the IPv6 link-local address will appear, hence we should set ipv6 as enabled also in order to pass the verification stage.

Reply:
That is a bug, and it is not related to this PR changes. If you set it to `false` as a temporary fix, I would understand.